packages/lesswrong/components/tagging/SubforumSubscribeSection.tsx

```diff
@@ -59,13 +59,31 @@ const SubforumSubscribeSection = ({
       flash({messageString: error.message});
     }
   }
+  
+  const onUnsubscribe = async (e: React.MouseEvent<HTMLButtonElement>) => {
+    try {
+      e.preventDefault()
+
+      captureEvent('subforumUnsubscribeClicked', {tagId: tag._id})
+
+      if (currentUser) {
+        void updateCurrentUser({profileTagIds: currentUser.profileTagIds.filter(id => id !== tag._id)})
```

Comment:
If we're attempting to catch errors (from attempting to unsubscribe), this should probably be `await` rather than `void`?